sync/upstream.py

```diff
@@ -967,5 +967,8 @@ def update_pr(git_gecko, git_wpt, sync, action, merge_sha=None, base_sha=None):
             if sync.status not in ("complete", "wpt-merged"):
                 env.bz.comment(sync.bug, "Upstream PR merged")
                 sync.finish("wpt-merged")
+            elif (sync.status == "wpt-merged" and
```

Comment:
Why don't we just do this when we call `merge_pr` rather than waiting for the event?

Reply:
I did consider this, will move it over to that part of the logic.